apollo-client/src/main/java/com/ctrip/framework/apollo/Config.java

```diff
@@ -181,6 +181,17 @@
    */
   public void addChangeListener(ConfigChangeListener listener, Set<String> interestedKeys);
 
+  /**
+   * Add change listener to this config instance, will only be notified when any of the interested keys is changed in this namespace.
+   *
+   * @param listener the config change listener
+   * @param interestedKeys the keys that the listener is interested in
+   * @param interestedKeyPatterns the key patterns that the listener is interested in
+   *
+   * @since 1.0.0
```

Comment:
@since 1.3.0?

Reply:
for `interestedKeyPatterns`, it may be better to provide some example

Reply:
Sure! I've added some unit tests, and if needed, I'll add some documentation too. BTW, where should I add the examples? I have no permissions to edit the wiki.

Reply:
normally we would add it to the java doc, e.g.

`@param interestedKeyPatterns the key patterns that the listener is interested in, e.g. xxx, which means yyy`

Reply:
Done as suggested. If any further changes should be done, please let me know.

Reply:
Thanks, will take a detailed look soon.